services-core/src/gas_price/gnosis_safe.rs

```diff
@@ -121,9 +116,10 @@ fn estimate_with_limits(
 
 #[cfg(test)]
 pub mod tests {
+    use super::super::tests::TestTransport;
     use super::super::DEFAULT_GAS_LIMIT;
     use super::*;
-    use crate::util::FutureWaitExt as _;
+    use crate::util::FutureWaitExt;
```

Comment:
I don't understand this change, as I don't see `FutureWaitExt` used anywhere. The `as _` is used in Rust especially for traits to import them into scope so you can use the methods on implementations but when you don't actually need to reference the trait directly (either implementing it or adding a generic trait bound). I slightly prefer it as it was before.